crates/sui-node/src/lib.rs

```diff
@@ -2042,6 +2051,30 @@ impl SuiNode {
     }
 }
 
+enum ValidatorGrpcServerHandle {
```

Comment:
this enum seems to have nothing (internally) to do with being a validator gRPC server? is this just to make starting idempotent? I am confused by its name being so much more specific than what it appears to do

Reply:
Fair... but i'm not sure what else to call it? its not really a general purpose thing?

Reply:
I like SpawnOnce